main.py

```diff
@@ -1,10 +1,35 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as config_file:
+        players_data = json.load(config_file)
+
+    for player, data in players_data.items():
+
+        race = Race.objects.get_or_create(data["race"]["name"],
```

Comment:
The `get_or_create` method requires the first argument to be a field lookup, such as `name=data['race']['name']`, instead of just `data['race']['name']`. This applies to both the `Race` and `Guild` objects.